zcash_client_backend/CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- `zcash_client_backend::proposal::ProposalError`:
+  - `SendsMemoToTransparentRecipient`: signals an attempt to send a memo to a transparent recipient
+  - `ResultingPaymentInvalid(Zip321Error)`: details the ZIP-321 error that the proposal 
+  resulting causes.
+- `zcash_client_backend::data_api::wallet`:
+  - `SpendableNotes::total`: returns the sum of all spendable notes in `Zatoshis`
+  - `propose_send_max_transfer`: calls `propose_send_max` for a given `InputSelector`
```

Comment:
The CHANGELOG entry for an addition should describe the purpose, not the implementation.